src/v2/ion/RemediationConstants.js

```diff
@@ -24,6 +24,10 @@ const FORMS = {
   SELECT_AUTHENTICATOR_ENROLL_DATA: 'select-authenticator-enroll-data',
   AUTHENTICATOR_ENROLLMENT_DATA: 'authenticator-enrollment-data',
   ENROLL_AUTHENTICATOR: 'enroll-authenticator',
+  ENROLL_SELECT_CHANNEL: 'enroll-with-another-channel',
```

Comment:
- `ENROLL_SELECT_ANOTHER_CHANNEL`?

Reply:
The reason I wanted to use ENROLL_SELECT_CHANNEL is because we are using this as either switch or select in mobile.

Reply:
sure then we'd rename the View as well. (EnrollSeectChannelOktaVerify)

Reply:
Then it becomes specific to OV. I thought we are going to have generic names and filter using authenticator type.

Reply:
SELECT_ENROLL_CHANNEL : 'select-enrooll-channel'

Reply:
Updated to select-enrollment-channel as we decided :)